### PR TITLE
Issue#76 fe event listing mobile

### PR DESCRIPTION
**Pull request by mariagluck**
I have cleaned the navbar (so brought out components), since there were a lot of conditionals. Now if you click on something that requires login and you are not, you will be redirected. mobile and desktop.

### CODE REVIEW

File: src/components/CreateListingPage/Form.jsx [hunk 1]
```diff
@@ -268,6 +268,7 @@ export default function Form(){
                         name="summary"
                         onChange={changeHandler}
                     />
+                    <p className="characters-limit">0/100</p>
```
Comment (Mxx1029) [inline src/components/CreateListingPage/Form.jsx:271]: is this supposed to count the characters? That's how i would understand it as from a user's point of view. as there's no logic behind it, that would make it count, maybe just say "max. 100"?

File: src/components/NavBar/NavBar.jsx [hunk 2]
```diff
@@ -1,45 +1,39 @@
...
 import "./NavBar.scss";
 import logo from "../../images/what toDo.png";
-import {FaSearch, FaHeart, FaRegHeart,FaSignInAlt, FaSignOutAlt} from "react-icons/fa";
+import { FaSearch } from "react-icons/fa";
 
-export default function NavBar(){
+export default function NavBar() {
   const [modal, setModal] = useState(false);
-  const Toggle = () => setModal(!modal);//to open mobile search modal
+  const Toggle = () => setModal(!modal); //to open mobile search modal
```
Comment (Mxx1029) [inline src/components/NavBar/NavBar.jsx:12]: is Toggle a component or a class? Or only a function? If only a function, better keep it lowercase maybe?

File: src/components/NavBar/NavBarLogged.jsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+      <Link to={"/create-listing"}>
+        <FaRegEdit className="create-event-mobile-icon" />
+      </Link>
+
+      <Link to={"/event"}>
+        <FaSignOutAlt className="login-icon" />
+      </Link>
+
+      <Link to={"/whishlist"}>
```
Comment (Mxx1029) [inline src/components/NavBar/NavBarLogged.jsx:30]: typo here :)

File: src/components/NavBar/NavBar.jsx [hunk 4]
```diff
@@ -1,45 +1,39 @@
 import React, { useState } from "react";
-import { useNavigate, Link} from "react-router-dom";
+import { Link } from "react-router-dom";
+import NavBarNotLogged from "./NavBarNotLogged";
```
Comment (Mxx1029) [inline src/components/NavBar/NavBar.jsx:3]: i wonder if Logged/NotLogged is the best name choice here. As Logging is so much associated with console.log().
what about NavBarUser / NavBarNoUser or NavBarGuest.
Or NavBarLoggedIn / NavBarNotLoggedIn
Or we just keep it, I actually don't mind, just a thought here :) You decide !

**Review by Mxx1029 — APPROVED** [review]
Great work! I'm really excited to see how it's all coming along 💯 
Some minor comments :)

File: src/components/NavBar/NavBar.jsx (hunk 2 above)
Comment (mariagluck) [reply src/components/NavBar/NavBar.jsx:12]: this is a brain fart I guess. it should be toggle no caps.

File: src/components/CreateListingPage/Form.jsx (hunk 1 above)
Comment (mariagluck) [reply src/components/CreateListingPage/Form.jsx:271]: mm ok, I was following the design but of course it makes sense it is a word counter or other option, like max,,,

File: src/components/NavBar/NavBarLogged.jsx (hunk 3 above)
Comment (mariagluck) [reply src/components/NavBar/NavBarLogged.jsx:30]: typo and brainfart

File: src/components/NavBar/NavBar.jsx (hunk 4 above)
Comment (mariagluck) [reply src/components/NavBar/NavBar.jsx:3]: I can use the user option. I wrote initially NavBarLoggedIn / NavBarNotLoggedIn, it felt extremely long and honestly too redundant or something. ...i will use the user option I guess

Comment (Mxx1029) [reply src/components/NavBar/NavBar.jsx:3]: sounds good to me :)

**Pull request merged**